frontend/.gitignore

```diff
@@ -0,0 +1,21 @@
+# See https://help.github.com/ignore-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
```

Comment:
all this `misc` ones, with `.DS_Store` and `.env.*` are already in the main `.gitignore`